server/server.js

```diff
@@ -11,12 +11,28 @@ app.use(express.static(path.join(__dirname + '/../public')));
 
 const port = 4000;
 
-app.get('/', (req, res) => {
-  console.log('in server get req');
-  res.send("hello")
-
+app.get('/listing/:productId', (req, res) => {
+  const qString = `SELECT shopId FROM products WHERE id=
+  ${req.params.productId}`;
+  db.query(qString, (err, shopId) => {
+    if (err) {
+      console.log('Error getting shopId', err);
+    }
+    return db.query(`SELECT * FROM shops WHERE id=${shopId[0].shopId}`, (err, shopInfo) => {

```

Comment:
What's the purpose of the return here and just below? 
This ought to work without using those since you're using error callback style - just call the query and  the function ends when you either throw an error out of it, or send the response.